src/components/Upload.tsx

```diff
@@ -19,18 +25,18 @@ function Upload({ disableConfig }: UploadProps) {
   const columns = useDataStore((state) => state.columns);
   const dataDictionary = useDataStore((state) => state.uploadedDataDictionary);
   const uploadedDataTableFileName = useDataStore((state) => state.uploadedDataTableFileName);
-  const loadConfigOptions = useDataStore((state) => state.loadConfigOptions);
-  const configOptions = useDataStore((state) => state.configOptions);
-  const selectedConfig = useDataStore((state) => state.selectedConfig);
-  const setSelectedConfig = useDataStore((state) => state.setSelectedConfig);
+  const { appFetchesConfigOptions } = useFreshDataActions();
+  const configOptions = useConfigOptions();
+  const selectedConfig = useConfig();
+  const { userSelectsConfig } = useFreshDataActions();
```

Comment:
I like how you organize it by topic, rather than expanding both actions once. Should we add that to our best practices: https://neurobagel.github.io/dev-docs/frontend/Zustand/ ?